Bake.py

```diff
@@ -1532,6 +1536,13 @@ def execute(self, context):
         # Get tilenums
         tilenums = UDIM.get_tile_numbers(objs, self.uv_map) if self.use_udim else [1001]
 
+        # Enable disabled layers if needed
+        disabled_layers = None
```

Comment:
Should this be an empty list instead of None? Because it looks like it will be iterated later. It's just for a safety purpose.